hedera-mirror-rest/api/v1/openapi.yml

```
@@ -2036,6 +2059,67 @@ components:
       type: array
       items:
         $ref: '#/components/schemas/NetworkFee'
+    NetworkStake:
+      type: object
+      required:
+        - max_staking_reward_rate_per_hbar
+        - node_reward_fee_fraction
+        - stake_total
+        - staking_period
+        - staking_period_duration
+        - staking_periods_stored
+        - staking_reward_fee_fraction
+        - staking_reward_rate
+        - staking_start_threshold
+      properties:
+        max_staking_reward_rate_per_hbar:
+          description: The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day
+          type: integer
+          format: int64
+        node_reward_fee_fraction:
+          description: The fraction of the network and service fees paid to the node reward account 0.0.801
+          type: number
+          format: int64
+        stake_total:
+          description: The total amount staked to the network in tinybars the start of the current staking period
+          type: number
+          format: int64
+        staking_period:
+          allOf:
+            - $ref: '#/components/schemas/TimestampRange'
+            - description: The timestamp range of the staking period
+        staking_period_duration:
+          description: The number of minutes in a staking period
+          type: number
+          format: int64
+        staking_periods_stored:
+          description: The number of staking periods for which the reward is stored for each node
+          type: number
+          format: int64
+        staking_reward_fee_fraction:
+          description: The fraction of the network and service fees paid to the staking reward account 0.0.800
+          type: number
+          format: int64
```

Change both fractions to:
```suggestion
          format: float
```

---

These should stay as type=number. See swagger docs https://swagger.io/docs/specification/data-models/data-types/#numbers